_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -2409,174 +2476,304 @@
 /obj/structure/cable{
 	icon_state = "4-8"
 	},
-/obj/machinery/door/firedoor/border_only{
-	dir = 8
-	},
-/obj/machinery/door/firedoor/border_only{
-	dir = 4
+/turf/open/floor/plasteel/white,
+/area/ship/hallway/fore)
+"TH" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/machinery/portable_atmospherics/canister/oxygen,
```

Comment:
You should probably give them a handful of spare plasmaman tanks in a locker somewhere in cargo so plasmemes aren't even more SOL than usual. They don't need a canister, but like two or three spare internal tanks wouldn't go amiss.